foosballv2s/MainActivity.cs

```diff
@@ -26,10 +29,23 @@ protected override void OnCreate(Bundle savedInstanceState)
         [Export("SubmitTeamNames")]
         public void SubmitTeamNames(View view)
         {
-            //TODO: validate team names with Regex
-            Intent intent = new Intent(this, typeof(BallImageActivity));
-            StartActivity(intent);
+            Validator v = new Validator();
+            mteam1Name = (EditText)FindViewById(Resource.Id.team1Name);
```

Comment:
We'll have to resolve conflicts, as the type is being changed to AutoCompleteTextView.